pydrive2/auth.py

```diff
@@ -396,38 +390,58 @@ def LoadCredentialsFile(self, credentials_file=None):
         :raises: InvalidConfigError, InvalidCredentialsError
         """
         if credentials_file is None:
-            self._default_storage = self._storages["file"]
+            self._default_storage = self._storage_registry["file"]
+            credentials_file = self.settings.get("save_credentials_file")
             if self._default_storage is None:
                 raise InvalidConfigError(
                     "Backend `file` is not configured, specify "
                     "credentials file to read in the settings "
                     "file or pass an explicit value"
                 )
         else:
-            self._default_storage = Storage(credentials_file)
+            self._default_storage = FileBackend()
```

Comment:
information is lost at this point that this storage is from the specified `credentials_file`. I'm not 100% understand the use case why would someone do `auth.LoadCredentialsFile("some/custom/path")`, but it looks like we are changing the behavior here ...

Reply:
okay, before it was important since the credentials object was triggering save on refresh, not sure again yet if this is important here ...